scripts/solutions/clean_up.sh

```diff
@@ -18,4 +18,4 @@ find problems -name '*.eprime-infor'  -delete
 
 # the eprimes are duplicated during solving. remove them here.
 rm -f problems/*/conjure-mode/*/savilerow-mode/*/solver/*/*.eprime
-rm -rf slurm
+rm -rf slurm/sh
```

Comment:
did you intend to have a * here?

Reply:
no. It should create a sub-directory with only the sh files to make the clean-up easier: we will have 3 directories: one for the sh, one for the stdout and one for the stderr files

Reply:
oh ok, didn't notice that also changed.

Reply:
fyi, I had to make another edit: https://github.com/conjure-cp/EssenceCatalog-runs/commit/83221dc693552b0f2711910a21119f0f60ad9d9a